plugins/system/apticron

```diff
@@ -0,0 +1,228 @@
+#!/bin/bash
+# -*- sh -*-
+: <<=cut
+
+=head1 NAME
+
+apticron - Show the result of I<"apt-get -u dist-upgrade"> with four different graphs
+
+=head1 APPLICABLE SYSTEMS
+
+All systems with "apt-get" and "grep"
+
+=head1 CONFIGURATION
+
+The following is the default configuration
+
+  [apticron]
+  user root
+
+You could define two alert levels and the graph language
+
+  [apticron]
+  env.warning          [value]        (default:  80)
+  env.critical         [value]        (default: 100)
+  env.language         [en|de|es]     (default:  en)
+
+or if you like define more details
+
+  env.warning_update   [value]        (default:  80 or env.warning, if set)
+  env.warning_newone   [value]        (default:  25% of env.warning_update)
+  env.warning_remove   [value]        (default:   2% of env.warning_update)
+  env.warning_holdit   [value]        (default:   2% of env.warning_update)
+
+  env.critical_update  [value]        (default: 100 or env.critical, if set)
+  env.critical_newone  [value]        (default:  25% of env.critical_update)
+  env.critical_remove  [value]        (default:   2% of env.critical_update)
+  env.critical_holdit  [value]        (default:   2% of env.critical_update)
+
+=head1 DESCRIPTION
+
+This plugin is a graphic version of die I<apicron package>, but it sends no mails.
+
+It showns graphs for
+
+  - upgraded
+  - newly installed
+  - to remove
+  - not upgraded
+
+The plugin itself can be stored in any directory.
+A symbolic link must be createdin the /etc/munin/plugins/ directory as follows: 
```

Comment:
The use of symlinks in /etc/munin/plugins/ is Debian-specific.
You do not need to describe here, how munin plugins are installed. This is a task for package maintainers or the general munin documentation.